docs/markdown/_include_qt_base.md

```diff
@@ -49,6 +54,11 @@ It takes no positional arguments, and the following keyword arguments:
   - `dependencies`: dependency objects whose include directories are used by moc.
   - `include_directories` (string | IncludeDirectory)[]: A list of `include_directory()`
     objects used when transpiling the .moc files
+  - `preserve_paths` bool: *New in 1.4.0*. If `true`, specifies that the output
```

Comment:
I feel like we typically use `Since` and not `New in`. Are you matching something else?

Reply:
This file is full of `New in ...`, but `Since` is good for me!

Reply:
Oh I was trying to say if you're being consistent, that is good with me. But whatever you want.